src/components/Pressable/PressableWithFeedback.js

```diff
@@ -39,25 +39,22 @@ const PressableWithFeedbackDefaultProps = {
 
 const PressableWithFeedback = forwardRef((props, ref) => {
     const propsWithoutWrapperStyles = _.omit(props, omittedProps);
-    const [disabled, setDisabled] = useState(props.disabled);
-    const [isPressed, setIsPressed] = useState(false);
+    const [isPressed, setPressed] = useState(false);
+    const [isPressedIn, setPressedIn] = useState(false);
     const [isHovered, setIsHovered] = useState(false);
-
-    useEffect(() => {
-        setDisabled(props.disabled);
-    }, [props.disabled]);
+    const isDisabled = useMemo(() => props.disabled || isPressed, [props.disabled, isPressed]);
```

Comment:
We don't need useMemo here, the logic is simple enough to just recalculate it

Reply:
You're right. Updated!